src/pybind/mgr/dashboard/plugins/debug.py

```diff
@@ -55,14 +55,14 @@ def setup(self):
         self._refresh_health_checks()
 
     @no_type_check
```

Comment:
lemme try.

Reply:
no, otherwise I have 
```
File "ceph/src/pybind/mgr/.tox/mypy/lib/python3.9/site-packages/mypy/nodes.py", line 2660, in __getattribute__
    raise AssertionError(object.__getattribute__(self, 'msg'))
AssertionError: Var is lacking info
```
python3.6 has the same issue, and mypy 0.80 is able to finish the check. but it is not yet released at the time of writing.